src/main/java/org/candlepin/util/ContentOverrideValidator.java

```diff
@@ -130,6 +137,15 @@ public void validate(Collection<? extends ContentOverrideDTO> overrides) {
                         String.join(", ", invalidValues)));
                 }
 
+                if (!invalidLengthValues.isEmpty()) {
+                    if (builder.length() > 0) {
+                        builder.append('\n');
+                    }
+
+                    builder.append(i18n.tr("The following override values exceed the max {0} " +
+                        "character length: {1}", MAX_VALUE_LENGTH, String.join(", ", invalidLengthValues)));
+                }
```

Comment:
I figured we could add some better output for exceedingly long values.

Reply:
This is going to output a list of *very* long values, to the point where the error message itself might not be valuable as written. It's probably better to output the key here rather than the value, or at least, truncate the value to something more consumable.